wallet/wallet-address-generator-lib/src/lib.rs

```diff
@@ -68,11 +68,12 @@ pub struct CliArgs {
     pub network: Network,
 
     /// Number of addresses to generate and display
-    #[clap(long, default_value_t = 1)]
+    #[clap(long, short = 'n', default_value_t = 1)]
     pub address_count: u8,
 
     /// Mnemonic phrase (12, 15, or 24 words as a single quoted argument). If not specified, a new mnemonic phrase is generated and printed.
     #[clap(long)]
+    #[arg(hide = true)]
```

Comment:
this will hide it from displaying in help. But it still can be used, right? Is it good enough? just asking

Reply:
In a previous commit I removed it completely, but then found that it's used in an important test to verify address generation. So I decided hiding is good enough. Let's see how that goes.